horace_core/sqw/@sqw/refine_crystal_strip_pars.m

```diff
@@ -34,10 +35,12 @@
 angdeg = pars_in(end-5:end-3);
 rotvec = pars_in(end-2:end);
 
-% Compute rotation matrix and new ub matrix
-rotmat = rotvec_to_rotmat2(rotvec);
-ub = ubmatrix(xtal.urot,xtal.vrot,bmatrix(alatt,angdeg));
-rlu_corr = ub\rotmat*xtal.ub0;
+% Compute rotation matrix and new ub matrix : unclear meaning of this code.
+% it seems the replacemant below 
+%rotmat = rotvec_to_rotmat2(rotvec);
+%ub = ubmatrix(xtal.urot,xtal.vrot,bmatrix(alatt,angdeg));
+%rlu_corr = ub\rotmat*xtal.ub0;
```

Comment:
this makes the user even more uncertain. The sentence on L 39 is incomplete. Can we drop all the new comments and just let people look at crystal_alignment_info?

Reply:
finshed this comment. Hopefully it is coherent now